_posts/2019-08-15-release-1.9.0.md

```diff
@@ -0,0 +1,352 @@
+   FLIP-38.
+ * Flink can be compiled and run on Java 9. Note that certain components
+   interacting with external systems (connectors, filesystems, reporters) may
+   not work since the respective projects may have skipped Java 9 support.
+
+
+## Release Notes
+
+Please review the [release
+notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344601)
+for a more detailed list of changes and new features if you plan to upgrade
+your Flink setup to Flink 1.9.0.
+
+
+## List of Contributors
+
+We would like to thank all contributors who have made this release possible:
+
+Abdul Qadeer (abqadeer), Aitozi, Alberto Romero, Aleksey Pak, Alexander
+Fedulov, Alice Yan, Aljoscha Krettek, Aloys, Andrew Duffy, Andrey Zagrebin,
+Ankur, Artsem Semianenka, Benchao Li, Biao Liu, Bo WANG, Bowen L, Chesnay
+Schepler, Clark Yang, Congxian Qiu, Cristian, Danny Chan, David Moravek, Dawid
+Wysakowicz, Dian Fu, EronWright, Fabian Hueske, Fabio Lombardelli, Fokko
+Driesprong, Gao Yun, Gary Yao, Gen Luo, GitHub, Gyula Fora, Hequn Cheng,
+Hongtao Zhang, Huang Xingbo, HuangXingBo, Hugo Da Cruz Louro, Humberto
+Rodríguez A, Hwanju Kim, Igal Shilman, Jamie Grier, Jark Wu, Jason, Jasper
+Yue, Jeff Zhang, Jiangjie (Becket) Qin, Jiezhi.G, Jincheng Sun, Jing Zhang,
+Jingsong Lee, JingsongLi, Juan Gentile, Jungtaek Lim, Kailash Dayanand, Kevin
+Bohinski, Konstantin Knauf, Konstantinos Papadopoulos, Kostas Kloudas, Kurt
+Young, Lakshmi, Lakshmi Gururaja Rao, Leeviiii, LouisXu, Maximilian Michels,
+Nico Kruber, Niels Basjes, Paul Lam, PengFei Li, Peter Huang, Pierre Zemb,
+Piotr Nowojski, Piyush Narang, Richard Deurwaarder, Robert Metzger, Robert
+Stoll, Romano Vacca, Rong Rong, Rui Li, Ryantaocer, Scott Mitchell, Seth
+Wiesman, Shannon Carey, Shimin Yang, Stefan Richter, Stephan Ewen, Stephen
+Connolly, Steven Wu, SuXingLee, TANG Wen-hui, Thomas Weise, Till Rohrmann,
+Timo Walther, TisonKun, Tom Goong, TsReaper, Tzu-Li (Gordon) Tai, Ufuk Celebi,
+Victor Wong, WangHengwei, Wei Zhong, WeiZhong94, Xintong Song, Xpray,
+XuQianJin-Stars, Xuefu Zhang, Xupingyong, Yangze Guo, Yu Li, Yun Gao, Yun
+Tang, Zhanchun Zhang, Zhenghua Gao, Zhijiang, Zhu Zhu, Zi Li, ZiLi Chen, Zili
+Chen, aloys, ap, arganzheng, azagrebin, bd2019us, beyond1920, biao.liub,
+blueszheng, boshu Zheng, bowen.li, chenqi, chummyhe89, chunpinghe, dcadmin,
+dianfu, godfrey he, godfreyhe, guanghui01.rong, hehuiyuan, hello, hequn8128,
+ifndef-SleePy, jackyyin, joongkeun.yang, klion26, lamber-ken, leesf, liguowei,
+lincoln-lil, liyafan82, luoqi, mans2singh, maqingxiang, maxin, mjl, okidogi,
+ozan, potseluev, qiangsi.lq, qiaoran, robbinli, shaoxuan-wang, shengqian.zhou,
+shenlang.sl, shuai-xu, sunhaibotb, sunjincheng121, tianchen, tianchen92,
+tison, tom_gong, vinoyang, vthinkxie, wanggeng3, wenhuitang, winifredtamg,
+xl38154, xuefuz, xuyang1706, yangfei5, yanghua, yuzhao.cyz, zentol,
+zhangxin516, zhangxinxing, zhaofaxian, zhijiang, zhuzhu.zz, zjuwangg, 云邪, 林小铂,
+黄培松, 姬平, 时无两丶.
```

Comment:
One could try to deduplicate this list. I can see TisonKun appearing 4 times (TisonKun, Zi Li, ZiLI Chen, Zili Chen).

Reply:
From what I know, "云邪" is "Jark Wu" and "姬平" is "Xupingyong".

Reply:
It seems we extracted at least contributors' Github id and name.

Reply:
The duplicated ones I can identify are {Zhu Zhu, zhuzhu.zz}, {biao.liub, ifndef-SleePy},{Chesnay
Schepler, zentol}, {Xuefu Zhang, xuefuz}.
Extracting the contributors' display name and user name from commit, as yanghua mentioned, can help to avoid duplication. But we still need some extra efforts to revise it. Some people's git display name are not very formal compared to their JIRA name.

Reply:
Thanks. I've deduplicated based on your suggestions.